support/src/test/java/com/shopify/graphql/support/NoMutationSchemaIntegrationTest.java

```diff
@@ -0,0 +1,103 @@
+package com.shopify.graphql.support;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class NoMutationSchemaIntegrationTest {
```

Comment:
This is a lot of duplication just to test the case where there are no mutations.